core/src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/StackSyncService.java

```diff
@@ -226,11 +233,21 @@ private void syncDeletedInstance(Stack stack, Map<InstanceSyncState, Integer> in
         }
     }
 
-    private void handleSyncResult(Stack stack, Map<InstanceSyncState, Integer> instanceStateCounts, boolean stackStatusUpdateEnabled) {
+    private void handleSyncResult(Stack stack, Map<InstanceSyncState, Integer> instanceStateCounts, boolean stackStatusUpdateEnabled, boolean cmServerRunning) {
```

Comment:
Getting quite a lot of control booleans. We could extract this to a SyncConfig class or something similar.